src/SFA.DAS.RoatpGateway.Web/Services/GatewayOrganisationChecksOrchestrator.cs

```diff
@@ -53,10 +58,15 @@ public async Task<LegalNamePageViewModel> GetLegalNameViewModel(GetLegalNameRequ
 
                 if (ukrlpDetails.VerifiedbyCharityCommission)
                 {
-                    var charityCommissionDetails = await _applyApiClient.GetCharityCommissionDetails(request.ApplicationId);
```

Comment:
Why change the source of information here? I am assuming that apply service is retrieving information from application data. Is there a change in this assumption?

Reply:
The story is to 'Replace old api call with outer api call'.  If we're just getting it from apply, there is no work needed in gateway at all, so I'm not sure what you mean. We may need to discuss.

Reply:
This is to get the stuff from 'charity' that we compare the details from charity with the details stored in apply (which is legal name, organisation status, and trustee information)